manifest.yml

```diff
@@ -32,7 +32,7 @@ dependencies:
       path: "FreeRTOS-Plus/Source/AWS/sigv4"
 
   - name: "coreHTTP"
-    version: "v2.1.0"
+    version: "7fbd7c3"
```

Comment:
What changes is this for?

Reply:
This includes the lhttp version of coreHTTP. Without it, the coreHTTP demos do not build.